### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by royr5**

### CODE REVIEW

File: db/data/test-data/index.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
-exports.articleData = require('./articles.js');
-exports.commentData = require('./comments.js');
-exports.topicData = require('./topics.js');
-exports.userData = require('./users.js');
+exports.articleData = require("./articles.js");
+exports.commentData = require("./comments.js");
+exports.topicData = require("./topics.js");
+exports.userData = require("./users.js");
+
+exports.indexTest = {
```
Comment (two-triangles) [inline db/data/test-data/index.js]: You have essentially duplicated what is already happening in this file - these changes should be removed and the file should be put back to how it looked originally.

File: db/data/development-data/index.js [hunk 2]
```diff
@@ -2,3 +2,11 @@ exports.articleData = require('./articles.js');
 exports.commentData = require('./comments.js');
 exports.topicData = require('./topics.js');
 exports.userData = require('./users.js');
+
```
Comment (two-triangles) [inline db/data/development-data/index.js:5]: You have essentially duplicated what is already happening in this file - these changes should be removed and the file should be put back to how it looked originally.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+# Instructions
+
+To clone this project and run it locally: 
+You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the correct database name for that environment.
```
Comment (two-triangles) [inline README.md]: You need to be explicit in telling the user, what each the values of PGDATABASE should be for each context.

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the changes Roshan, all good to merge! 👍

**Pull request merged**